views/pages/index.ejs

```diff
@@ -34,7 +34,7 @@
                     <tr<% if(index < 6){ %> class="bg-success" <% } %>> 
                         <th scope="row"><%= index+1 %></th>
                         <td><%= student.fullname %></td> 
-                        <td><%= student.acmpId %></td> 
+                        <td><a href=<%="https://acmp.ru/?main=user&id=" + student.acmpId %>><%= student.acmpId %></a></td> 
```

Comment:
Probably, static part of url can be moved out from EJS part. I mean this url part - "https://acmp.ru/?main=user&id=".